Sources/SwiftKuery/QueryResult.swift

```diff
@@ -92,6 +105,8 @@ public enum QueryResult {
                 }
                 return dictionary
             }
+            // As this function consumes the entire result set we can explicitly close it
```

Comment:
As this function consumes the entire result set we can explicitly close the result set allowing the release of the underlying connection.